bot/exts/moderation/modlog.py

```diff
@@ -522,14 +521,32 @@ async def log_cached_deleted_message(self, message: discord.Message) -> None:
                 f"**Message ID:** `{message.id}`\n"
                 f"**Sent at:** {format_dt(message.created_at)}\n"
                 f"[Jump to message]({message.jump_url})\n"
-                "\n"
             )
 
+        # If the message is a reply, add the reference to the response
+        if message.reference is not None and message.reference.resolved is not None:
+            resolved_message = message.reference.resolved
+
+            if isinstance(resolved_message, discord.DeletedReferencedMessage):
+                # Reference is a deleted message
+                reference_line = "**In reply to:** Deleted Message"
```

Comment:
I feel like you should still include the message author as well as the message link/message ID- and put "Deleted Message" in parenthesis.

Reply:
@vivekashok1221 
`discord.DeletedReferencedMessage` don't have any available attribute for `author`, I'm not sure how to get the author in such a case
`docs` : [discord.DeletedReferencedMessage](https://discordpy.readthedocs.io/en/stable/api.html?highlight=discord%20deletedreferencedmessage#discord.DeletedReferencedMessage)

Reply:
Ah my bad! we can directly pass `author` as a variable!

Reply:
In that case I guess you could put the message ID. Mods can use this message ID to find the content and author of the deleted referenced message.

Reply:
Alright, thanks